### PR TITLE
[15.0][MIG] product_tier_validation: Migration to 15.0

### PR DESCRIPTION
**Pull request by kaynnan**
cc @marcelsavegnago 

#919

### CODE REVIEW

**Comment by marcelsavegnago** [comment]
@augustodinizl, when you have a moment, could you please do a functional review?

**Comment by kaynnan** [comment]
ping @OCA/product-maintainers

File: product_tier_validation/tests/test_product_tier_validation.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+from odoo.tests import common, tagged
+
+
+@tagged("-at_install", "post_install")
+class TestProductTierValidation(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        group_ids = cls.env.ref("base.group_system").ids
+        cls.test_user_1 = cls.env["res.users"].create(
```
Comment (bosd) [inline product_tier_validation/tests/test_product_tier_validation.py:13]: Is it possible to re-use the users created in the base_tier_validation module?

Can you disable the email password reset and tracking. ( Like in base module )

**Review by bosd — COMMENTED** [review]
Lgtm, code review. Minor comment.

**Comment by bosd** [comment]
Can you recreate a runvoat for the functional test

**Pull request closed without merging**